### DISCUSSION TITLE
what’s the function of each sw elasticsearch indces :sw_metrics-all sw_records-all sw_management sw_segment sw_zipkin_span sw_log sw_browser_error_log

### DISCUSSION THREAD

**Discussion by ms787277296** [apache/skywalking#11710] [Q&A] [answered]
when i choose es as storage, it seems that sw create some indces :sw_metrics-all
sw_records-all
sw_management
sw_segment
sw_zipkin_span
sw_log
sw_browser_error_log

I want to know the function of them

**Comment by wu-sheng** [comment, marked as answer]
We presents docs, https://skywalking.apache.org/docs/main/next/en/setup/backend/storages/elasticsearch/#elasticsearch

**Reply by wu-sheng** [reply]
Simply, you could paste the question on the Ask AI on skywalking.apache.org

**Reply by ms787277296** [reply]
thanks a lot
